src/Services/Expenses.services.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const { generateUniqueID } = require('../utils/IdGenerator');
+
+let expenses = [];
```

Comment:
Why do we need let in this case?
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.